labs/accessing-optimizely-event-data/README.md

```diff
@@ -0,0 +1,140 @@
+```
+
+## Get Help
+
+For usage instructions, use the `help` command:
+
+```sh
+$ oevents help
+```
+
+## Authenticating
+
+Enriched Event data is served via [Amazon S3](https://aws.amazon.com/s3/).  You can authenticate `oevents` to AWS in two ways:
+
+1. (Recommended) Providing your [Optimizely Personal Access Token](https://docs.developers.optimizely.com/web/docs/personal-token) via the `OPTIMIZELY_API_TOKEN` environment variable or the `--token` command line argument. `oevents` will acquire AWS credentials using the [Optimizely Authentication API](https://docs.developers.optimizely.com/optimizely-data/docs/authentication-api).
+
+2. Providing your AWS credentials directly. See the [AWS user guide](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html) for instructions.
+
+You can use the `oevents auth` command to acquire temporary AWS credentials:
+
+```sh
+  $ oevents auth --token <optimizely personal access token>
+
+  export AWS_ACCESS_KEY_ID=<key id>
+  export AWS_SECRET_ACCESS_KEY=<secret access key>
+  export AWS_SESSION_TOKEN=<session token>
+  export AWS_SESSION_EXPIRATION=1594953226000
+  export S3_BASE_PATH=s3://optimizely-events-data/v1/account_id=12345/
+```
+
+## Exploring your Enriched Event data
+
+[decisions]: https://docs.developers.optimizely.com/web/docs/enriched-events-export#section-decisions
+[conversions]: https://docs.developers.optimizely.com/web/docs/enriched-events-export#section-conversions
+
+Enriched Events are partitioned into two top-level datasets, [decisions] and [conversions].  Each of these dataset is partitioned by date and experiment (for decisions) or event type (for conversions).  
+
+You can use `oevents ls` to list the all of the experiments that produced decision data on a given date:
+
+```sh
+$ oevents ls --type decisions --date 2020-05-10
+
+                           PRE experiment=10676850402/
+                           PRE experiment=14386840295/
+                           PRE experiment=14821050982/
+                           PRE experiment=15117030650/
+                           PRE experiment=17517981213/
+                           PRE experiment=17535310125/
+                           PRE experiment=8997901009/
+```
+
+You can also use `oevents` to list all of the event types collected on a given day:
+
+```sh
+$ oevents ls --type events --date 2020-05-10
+
+                           PRE event=search_query/
+                           PRE event=search_results_click/
+                           PRE event=add_to_cart/
+                           PRE event=purchase/
+```
+
+## Downloading your Enriched Event data
+
+You can use `oevents load` to download your Enriched Event data in [Apache Parquet](https://parquet.apache.org/) format.  Command line arguments can be used to specify a progressively narrower subset of your data.
+
+```sh
+$ oevents load --output ~/optimizely_data
+```
+
+will download *all* enriched event data associated with your Optimizely account.
```

Comment:
nit: I would personally structure this with the english above the code:

"The following will download *all* enriched event data associated with your Optimizely account:"

```sh
$ oevents load --output ~/optimizely_data
```

Which I think is more of a natural top down reading style.

Reply:
Done.  Good suggestion.